example_data/sample_bundle.json

```diff
@@ -470,34 +533,77 @@
             ]
           }
         ],
-        "valueString": "Next generation sequence analysis indicates that General Manager is heterozygous for the MECP2 c.953G>C p.(Arg318Pro) likely pathogenic variant that has been confirmed by Sanger sequence analysis"
+        "effectiveDateTime": "2020-10-22",
+        "note": [
+          {
+            "authorString": "genomic interpretation",
+            "text": "Next generation sequence analysis indicates that Maria321 BrighGosh321 is heterozygous for the SMC1A c.1263del p.(Lys422Serfs*28) pathogenic variant that has been confirmed by Sanger sequence analysis (see technical information below)."
+          }
+        ],
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "LA6577-6",
+              "display": "Negative"
+            }
+          ],
+          "text": "report finding"
+        }
```

Comment:
so overall positive or negative makes sense, I'm assuming it's this answer list that we're using it from <https://loinc.org/LL2431-6/>?

Reply:
yes